packages/visual-editor/src/utils/treeHelpers.ts

```diff
@@ -7,6 +7,12 @@ export function updateNode(
   node: CompositionComponentNode
 ) {
   if (node.data.id === nodeId) {
+    console.log(`:::updateNode() updating node.data`, {
+      oldData: node.data,
+      newData: updatedNode.data,
+      isShallowEqual: node.data === updatedNode.data,
+      isDeepEqual: isEqual(node.data, updatedNode.data),
+    });
```

Comment:
Do we want to keep this log? If so, I'd love to align this with the other logs as well
```suggestion
    console.debug(`[exp-builder.visual-editor::updateNode] updating node.data`, {
      oldData: node.data,
      newData: updatedNode.data,
      isShallowEqual: node.data === updatedNode.data,
      isDeepEqual: isEqual(node.data, updatedNode.data),
    });
```